_data/workinggroups.yaml

```diff
@@ -0,0 +1,21 @@
+- name: CI Working Group
+  url: meetings/CI-workinggroup
+  status: Active
+- name: Diversity and Inclusion Working Group
+  url: https://lists.aswf.io/g/diversity
+  status: Active
+- name: Rust Working Group
+  url: https://github.com/vfx-rs/organization
+  status: Active
+- name: Universal Scene Description Working Group (USD WG)
+  url: https://wiki.aswf.io/display/WGUSD/USD+Working+Group
+  status: Active
+- name: Assets Working Group
+  url: https://wiki.aswf.io/display/ARW
+  status: Archived
+- name: Python 3 Working Group
+  url: https://github.com/AcademySoftwareFoundation/wg-python3
+  status: Archived
+- name: Review and Approval Working Group
+  url: https://wiki.aswf.io/display/PRWG
+  status: Archived
```

Comment:
More generally, would there be a way to add link to Slack channels in WG descriptions?

Reply:
Yes - good idea. Added them as well as mailing lists, and have them listed with projects on the 'Getting Started' page.